src/synology_dsm/api/__init__.py

```diff
@@ -1,10 +1,15 @@
 """Synology API models."""
 
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from synology_dsm import SynologyDSM
+
 
 class SynoBaseApi:
     """Base api class."""
 
-    def __init__(self, dsm):
+    def __init__(self, dsm: "SynologyDSM") -> None:
```

Comment:
uhhh ... very nice 👍

Reply:
I added it because I was doing some right clicking and following the code, and it made it a lot easier

Reply:
It might be better to make it a relative import in the future, but probably fine either way for now